changelog.toml

```diff
@@ -0,0 +1,8 @@
+commit_types = [
+    { message = "^add", group = "Added"},
+    { message = "^remove", group = "Removed"},
+    { message = "^change", group = "Changed"},
+    { message = "^fix", group = "Bug Fixes"},
```

Comment:
```suggestion
    { message = "^add", group = "Added"},
    { message = "^remove", group = "Removed"},
    { message = "^change", group = "Changed"},
    { message = "^fix", group = "Bug Fixes"},
    { message = "^doc", group = "Documentation"},
    { message = "^refactor", group = "Refactor"},
    { message = "^test", group = "Testing"},
```

Reply:
We do not want test, refactor, and doc in the release data, thus no. :D